app/src/main/java/com/devrel/android/fitactions/home/FitStatsAdapter.kt

```diff
@@ -75,7 +75,7 @@ class FitStatsAdapter : ListAdapter<FitActivity, FitStatsAdapter.ViewHolder>(DIF
                 R.string.stat_date,
                 day,
                 calendar.get(Calendar.DAY_OF_MONTH),
-                calendar.get(Calendar.MONTH)
+                calendar.get(Calendar.MONTH) + 1
```

Comment:
Good catch. What are your thoughts of using simpledateformat instead? Thanks!

var month = SimpleDateFormat("MM")

month.format(calendar.time).toInt()

Reply:
Yes, I agree your suggestion. We can give expression the intent to display the month value using the `SimpleDateFormat` more clearly than using `+1`. I intend to modify the code soon. Thank you for your advice! :)

Reply:
I changed the code based on your advice: 0eee99c Could you review the diff again?